addons/udes_stock/models/stock_warehouse.py

```diff
@@ -101,6 +101,14 @@ def _domain_u_damaged_location_id(self):
         help="Allow users to inventory adjust reserved stock."
     )
 
+    u_disable_no_backorder_button_picking_type_ids = fields.Many2many(
```

Comment:
It may be worth adding a domain so you can only add picking types which are associated with the current warehouse

Reply:
Could you also update the readme for the new field